Gemfile

```diff
@@ -22,13 +24,16 @@ gem "sequel"
 
 group :cli, :development, :test do
   # Hanami testing helpers
-  gem "hanami-rspec"
+  gem "hanami-rspec", github: "hanami/rspec", branch: "main"
```

Comment:
just curious - why is this needed?

Reply:
@katafrakt It wasn't strictly necessary; there haven't been updates to that gem for the 2.1 cycle. But I was just taking the approach of making sure _everything_ was running on main, for consistency.